org.testeditor.web.backend.xtext/build.gradle

```diff
@@ -24,3 +24,7 @@ run {
     main mainClassName
     args 'server', configPath + 'config.yml'
 }
+
+dependencyCheck {
```

Comment:
is it possible to move this into the parent project (since both sub-projects have it)?

Reply:
no each sub project must specify its own threshold, whether sec checks should fail the build or not